packages/flutter/lib/src/material/stepper.dart

```diff
@@ -724,14 +775,21 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
   }
 
   Widget _buildVerticalBody(int index) {
+    final double? marginLeft = _stepIconMargin?.resolve(TextDirection.ltr).left;
+
     return Stack(
       children: <Widget>[
         PositionedDirectional(
-          start: 24.0,
+          // Adjusts the left side of the line so that it is centered when
+          // [StepperType.error] is displayed.
+          start: 24.0 + (marginLeft ?? 0.0),
```

Comment:
Here and a couple other places: If it's safe to assume start==left here, it would be useful to comment on why, specifically thinking about RTL environments.

It might also be useful to add a test specifically for RTL if there's a difference in behaviour.

Reply:
I added cleaner logic. Basically it will always center the lines when it is StepperType.vertical.